common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt

```diff
@@ -215,6 +224,28 @@ abstract class QrScanFragment : Fragment() {
 		}
 	}
 
+	open fun setZoom() {
```

Comment:
Why is this function open for overriding? It's only used in this class and can therefore be private and not open